### ISSUE TITLE
Automated testing

### ISSUE THREAD

**Issue by zzzwan** [marklogic/mlxprs#19]
Last-mile integration testing of the extension. This is complementary to the extensive back-end testing that happen as part of the MarkLogic release cycle. 

The minimum we should aim for is a single command that can run all of the tests in one shot. Next step is to hook up CI/CD so that it happens automatically on PRs and releases. (I’ll create a separate issue for that.)

If there are problems with the test library we‘re using we should investigate how others handle this, for example, the extensions that Microsoft delivers.

**Comment by zzzwan** [comment]
We already have QA testing covering js debugger functionalities at server side. However, automatic testing for the client/VS Code part is currently blocked by the above issue

**Comment by zzzwan** [comment]
Separated unit and integration test
